### PR TITLE
404s

### PR DESCRIPTION
**Pull request by kmherrmann**
fix a few 404s

### CODE REVIEW

**Review by fbolton — APPROVED** [review]
New links are working properly. LGTM.

File: docs/kratos/self-service/flows/user-settings.mdx [hunk 1]
```diff
@@ -279,7 +279,7 @@ example) to the configured settings URL.
   <TabItem value="oc">
 
 The Ory Network offers a default UI implementation. Visit
-[Bring Your Own UI](https://www.ory.sh/docs/guides/configure-ory-to-use-your-ui) to learn how to implement a custom UI.
+[Bring Your Own UI](https://www.ory.sh/docs/kratos/bring-your-own-ui/custom-ui-overview) to learn how to implement a custom UI.
```
Comment (aeneasr) [inline docs/kratos/self-service/flows/user-settings.mdx:282]: Add a redirect for this too?

File: docs/kratos/guides/deploy-kratos-example.mdx [hunk 2]
```diff
@@ -11,8 +11,7 @@ as reverse proxy, Digital Ocean as cloud provider, and the
 [Ory Kratos Node.js UI Reference](https://github.com/ory/kratos-selfservice-ui-node) as user interface. You can use another
 [relational database](https://www.ory.sh/docs/ecosystem/deployment#data-storage-and-persistence), a different reverse proxy,
 deploy on any other cloud host, and
-[spin up a custom interface in your favorite language](https://www.ory.sh/docs/kratos/sdk/01_overview.md) - this is just an
-example!
+[spin up a custom interface in your favorite language](https://www.ory.sh/docs/kratos/sdk/overview) - this is just an example!
```
Comment (aeneasr) [inline docs/kratos/guides/deploy-kratos-example.mdx:14]: Add a redirect for this too?

File: docs/kratos/self-service/flows/user-settings.mdx (hunk 1 above)
Comment (kmherrmann) [reply docs/kratos/self-service/flows/user-settings.mdx:282]: done

File: docs/kratos/guides/deploy-kratos-example.mdx (hunk 2 above)
Comment (kmherrmann) [reply docs/kratos/guides/deploy-kratos-example.mdx:14]: done

**Review by aeneasr — APPROVED** [review]

**Pull request merged**